bin/phpunit-each

```diff
@@ -11,7 +11,16 @@ $hasError = 0;
 
 chdir("$srcDir/tools");
 foreach ($args as $test) {
-  $cmd = sprintf("./scripts/phpunit --tap --log-junit %s %s %s",
+  if (file_exists("../Civi/Test/TAP.php")) {
+    $printCMD = "--printer '\Civi\Test\TAP'";
+  }
+  elseif (class_exists('PHPUnit_Util_Log_TAP')) {
```

Comment:
I'm confused - this is standalone PHP script, and it hasn't done any setup for autoloading -- so how would this check every return true?

Here's an alternative way to think about substituting `--tap` with `--printer \Civi\Test\TAP`: https://gist.github.com/totten/acfb8845dca00caef1420b78961139b5 . With the gist, the polyfill mechanism for TAP support goes entirely in `civicrm-core.git` (ie both the code `Civi/Test/TAP.php` and the conditional to activate it), so we don't have to worry about these edge-cases.

Reply:
Yeh ok can do that. Are you able to tac that onto my core TAP PR @totten?

Reply:
Sure, will do momentarily.